packages/core/src/parseAmplifyOutputs.ts

```diff
@@ -333,3 +335,21 @@ function getMfaStatus(
 
 	return 'off';
 }
+
+function mapNameToBucketInfo(
+	buckets: {
+		name: string;
+		bucket_name: string;
+		aws_region: string;
+	}[],
+) {
+	const mappedBuckets: Record<string, BucketInfo> = {};
+	buckets.forEach(({ name, bucket_name: bucketName, aws_region: region }) => {
+		mappedBuckets[name] = {
+			bucketName,
+			region,
+		};
+	});
```

Comment:
we could simplify this code,

```suggestion
return buckets.reduce<Record<string, BucketInfo>>((
mappedBuckets, 
{ name, bucket_name: bucketName, aws_region: region })=>{
  mappedBuckets[name] ={
  bucketName,
  region
  }
  return mappedBuckets;
}, {})

```

Reply:
I thought about reduce, i think this reads better :)